### DISCUSSION TITLE
Universal Links don't seem to be working for me on any of the latest public beta releases of iOS or iPadOS with the latest GitHub release from Testflight

### DISCUSSION THREAD

**Discussion by gitatmax** [community/community#2895] [Mobile]
Hey! I’ve enabled Universal Links and gone through the suggested process of long-pressing links on GitHub to select the “Open in GitHub” option; but even still, in GitHub for iPad the links in Notifications keep opening up in Safari. What might I be doing wrong, or is this how it’s supposed to work? Thanks. :)

**Comment by gabeio** [comment]
I had a similar problem when I changed the (Settings >) External Links from In-App to Safari. After I changed it back everything would open in the GitHub App correctly. Hope this helps!

**Reply by gitatmax** [reply]
Hey, I checked, and it looks like that's how I'm set up; appreciate it, though!![image](https://user-images.githubusercontent.com/7244793/101776420-e4196980-3abe-11eb-9e10-56ef4ccb0bfe.png)

**Reply by gabeio** [reply]
If you change that to In-App it should start working. At least it did for me.

**Reply by gitatmax** [reply]
Ah, I misunderstood you—duh! I'll give it a whirl. Thank you again

**Reply by gabeio** [reply]
No worries. Hope it helps!

**Reply by gitatmax** [reply]
Just to follow up here, it seems to be with just release notifications!
